crates/api_models/src/payments.rs

```diff
@@ -3325,6 +3325,8 @@ pub struct PaymentListFilterConstraints {
     pub limit: u32,
     /// The starting point within a list of objects
     pub offset: Option<u32>,
+    /// The amount to filter payments list
+    pub amount: Option<AmountFilter>,
```

Comment:
I feel this should be renamed to `amount_filter`. Generally `amount` should be a number.